### DISCUSSION TITLE
Get single Image Attachment from Data Provider (Budibase Internal Database) without using a Repeater

### DISCUSSION THREAD

**Discussion by iraklishubitidze** [Budibase/budibase#12036] [Show and tell]
Hey Peeps,
I was trying to get a single image attachment from the database without using a repeater and wanted to share the handlebars syntax for anyone interested. I have several "custom" images and wanted to save these in the database and not save them all elsewhere and get the link for every image.
In the docs I found the way, where you can use a Repeater to display the images from the database, but I wanted to avoid the repeater.

My example data:
![image](https://github.com/Budibase/budibase/assets/43496789/e6c906aa-f09a-42f0-a4ef-27ce5a4c9f1f)

If you want the the white globe in row 5, you can use the following structure:
![image](https://github.com/Budibase/budibase/assets/43496789/47f1ae51-8214-45ae-964a-68db37f8ad2d)

Connect the data provider to the correct database and give the data provider a name (in my case Image Data Provider).

Next you can use the following syntax in the URL of the image component, to get the image attachment from the fifth row:
{{ Image Data Provider.Rows.[5].ImageBlob.[0].url }}

Hope that helps some of you!

**Comment by melohagan** [comment]
Thanks for sharing @iraklishubitidze !

Incidentally there is also an open issue around making this a little more easy / obvious to do: https://github.com/Budibase/budibase/issues/8807
